lib/elastic_apm/spies/mongo.rb

```diff
@@ -61,7 +63,7 @@ def build_context(event)
           Span::Context.new(
             db: {
               instance: event.database_name,
-              statement: (collection && "#{collection}.#{event.command_name}"),
+              statement: event.command.inspect,
```

Comment:
Why `.inspect`?

Reply:
Otherwise it's a `Hash`. If I'm not mistaken, the `SpanSerializer` assumes the `db.statement` is a `String` [here](https://github.com/elastic/apm-agent-ruby/blob/master/lib/elastic_apm/transport/serializers/span_serializer.rb#L53).
Calling `#inspect` changes the `BSON::Document` to Extended JSON format.

Reply:
I should probably just call `#to_s` though...

Reply:
I've changed the line to use `event.command.to_s`